tx/textual/valuerenderer/enum_test.go

```diff
@@ -0,0 +1,24 @@
+package valuerenderer
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+
+	"cosmossdk.io/tx/textual/internal/testpb"
+)
+
+func TestFormatEnum(t *testing.T) {
```

Comment:
Can you also create an `enum.json` file following the format of other test files for testing?